Automattic/wp-calypso
# Migrate easy component styles to imports

#### Changes proposed in this Pull Request

* Migrates the easy blocks scss to webpack imports

#### Testing instructions

* Goto http://calypso.localhost:3000/devdocs/design
* Search for each component (button-group and emojify)
* Observe styles are correct

The other components have to be tested elsewhere.
